### PR TITLE
flight targets: pios_config: factor common

### PR DESCRIPTION
**Pull request by mlyle**
connects to #817

This attempts to make no functional change-- just to factor things
included common into one place.  Further audit is required on the
remaining pios_config lines for targets, but there's less to audit after
this.

### CODE REVIEW

**Comment by tracernz** [comment]
Looks good. `pios_flight_config` might be better than `pios_default_config`?

**Comment by mlyle** [comment]
now pios_flight_config

File: flight/PiOS/inc/pios_flight_config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#define PIOS_INCLUDE_FLASH_INTERNAL
+#define PIOS_INCLUDE_LOGFS_SETTINGS
+
+#define PIOS_INCLUDE_CHIBIOS
+
+// Resource limits
+#define CPULOAD_LIMIT_WARNING		80
+#define CPULOAD_LIMIT_CRITICAL		95
+#define IRQSTACK_LIMIT_WARNING		150
+#define IRQSTACK_LIMIT_CRITICAL		80
```
Comment (tracernz) [inline flight/PiOS/inc/pios_flight_config.h:39]: Tabs for alignment or variable-width stuff is a bad idea but I know this is copy-paste so meh. :wink:

**Review by tracernz — APPROVED** [review]
:+1:

**Comment by mlyle** [comment]
rebased. not sure why this one was conflicty

**Pull request merged**